subsys/bluetooth/shell/iso.c

```diff
@@ -181,11 +181,17 @@ static int cmd_cig_create(const struct shell *sh, size_t argc, char *argv[])
 			return -ENOEXEC;
 		}
 
-		param.interval = interval;
+		param.c_to_p_interval = interval;
+		param.p_to_c_interval = interval;
 	} else {
-		param.interval = 10000;
+		param.c_to_p_interval = 10000;
+		param.p_to_c_interval = 10000;
```

Comment:
Suggest to avoid duplicating literal values

Reply:
yes, working on a commit that sets the values separately, which fixes this